storage/storage-interface/src/state_store/state.rs

```diff
@@ -117,32 +125,47 @@ impl State {
             state_cache.next_version(),
         );
         // 3. `self` must be at a version equal or newer than the cache, because we assume
-        //    it is overlayed on top of the cache.
+        //    it is overlaid on top of the cache.
         assert!(self.next_version() >= state_cache.next_version());
 
         let overlay = self.make_delta(persisted);
+        let access_time_secs = duration_since_epoch().as_secs() as u32;
```

Comment:
should this be something else than the local current time?

Reply:
When we make it deterministic, will use the block time.

Reply:
also should be the same granularity as block time, not seconds, right?

Reply:
I used to think we write down in _ksecs but now I think that might not benefit in anyway except for saving some bytes. Can decide later when it matters.